src/main/java/com/redhat/devtools/lsp4ij/LanguageServersRegistry.java

```diff
@@ -162,7 +162,8 @@ private void loadServersAndMappingFromSettings() {
                                 launch.getUserEnvironmentVariables(),
                                 launch.isIncludeSystemEnvironmentVariables(),
                                 launch.getConfigurationContent(),
-                                launch.getInitializationOptionsContent()),
+                                launch.getInitializationOptionsContent(),
+                                launch.getClientConfigurationContent()),
```

Comment:
There's obviously going to be a proliferation of these types of changes where I've added `clientConfiguration` as another option of `UserDefinedLanguageServerDefinition` and its peer types/methods/etc.